components/Row/Row.styles.ts

```diff
@@ -5,9 +5,6 @@ export type ContentALignmentType = 'start' | 'center' | 'end' | 'stretch';
 
 export const root = (contentAlignment: ContentALignmentType) => contentAlignment === 'stretch' && '*:*:h-full';
 
-// TODO: Think about whether to finetune old flex width classes at the end
-// https://stanford.atlassian.net/browse/DS-1433
-
```

Comment:
After doing all the styles, it seems that using the width classes were ok. We kept that consistent everywhere that it's used so things are the same width and nothing looked weird